lib/test_queue/runner.rb

```diff
@@ -162,13 +168,34 @@ def summarize_internal
         puts @failures
       end
 
+      skipped_suites = @discovered_suites - run_suites
```

Comment:
I'm pretty sure we want to skip all this logic on remote masters (i.e., if `relay?` is true). Remote masters don't discover suites so `@discovered_suites` will always be empty, but `run_suites` will not be empty, so we'll always have "unexpected" suites. (If that turns out to be correct, it'd be great to have a test that verifies this if possible.)